### ISSUE TITLE
unbrowsable folder missing in quickstart 3.10.4

### ISSUE THREAD

**Issue by greend139** [Bedework/bedework#111]
Setting up a new install of quickstart 3.10.4 the unbrowsable folder in the public tree is not there. I started a quickstart 3.10.3 release and verified that it is there.  Is there an easy way to create this folder?  It does not appear possible through the admin UI.

**Comment by spartanicus** [comment]
Yes - that is an oversight which we'll check on for the next release.  You can follow the structure outlined here: https://wiki.jasig.org/display/BWK310/Create+your+calendar+hierarchy  Add the unbrowsable folder to /public and set access to "none" for all users/groups on the folder.  Then ensure that the underlying collections have access similar to /public/cals/MainCal.
